pkg/util/converter/prom.go

```diff
@@ -662,7 +666,7 @@ func readMatrixOrVectorMulti(iter *jsonitere.Iterator, resultType string, opt Op
 				Type:   data.FrameTypeTimeSeriesMulti,
 				Custom: resultTypeToCustomMeta(resultType),
 			}
-			if opt.Dataplane && resultType == "vector" {
+			if opt.Dataplane && resultType == "vector" || resultType == "scalar" {
```

Comment:
should the OR be wrapped in parentheses?
```suggestion
			if opt.Dataplane && (resultType == "vector" || resultType == "scalar") {
```